coverage/control.py

```diff
@@ -605,10 +605,10 @@ def _atexit(self, event="atexit"):
 
     def _on_sigterm(self, signum_unused, frame_unused):
         """A handler for signal.SIGTERM."""
+        signal.signal(signal.SIGTERM, self._old_sigterm)
         self._atexit("sigterm")
```

Comment:
this way if someone is furiously mashing ctrl+c they don't start multiple `self._atexit` calls

Reply:
might also be prudent to check if `self._atexit.__code__` is in the `frame_unused` currently